addons/link_tracker/tests/test_merge_utm_campaign.py

```diff
@@ -0,0 +1,91 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.tools import mute_logger
+from odoo.tests.common import TransactionCase
+
+
+class TestMergeUtmCampaign(TransactionCase):
+    def setUp(self):
+        result = super(TestMergeUtmCampaign, self).setUp()
+        self.UtmCampaign = self.env['utm.campaign']
+        self.LinkTracker = self.env['link.tracker']
+        self.LinkTrackerClick = self.env['link.tracker.click']
+        self.utm_source = self.env['utm.source'].create({
+            'name': 'utm source'
+        })
+        self.utm_medium = self.env['utm.medium'].create({
+            'name': 'utm medium'
+        })
+        self.utm_campaign_1 = self.UtmCampaign.create({
+            'name': 'utm campaign 1'
+        })
+        self.utm_campaign_2 = self.UtmCampaign.create({
+            'name': 'utm campaign 2'
+        })
+        self.utm_campaign_3 = self.UtmCampaign.create({
+            'name': 'utm campaign 3'
+        })
+        self.link_tracker_1 = self.LinkTracker.create({
+            'url': 'www.dodd.com',
+            'campaign_id': self.utm_campaign_1.id,
+            'medium_id': self.utm_medium.id,
+            'source_id': self.utm_source.id
+        })
+        self.link_tracker_2 = self.LinkTracker.create({
+            'url': 'www.dodd.com',
+            'campaign_id': self.utm_campaign_2.id,
+            'medium_id': self.utm_medium.id,
+            'source_id': self.utm_source.id
+        })
+        self.link_tracker_3 = self.LinkTracker.create({
+            'url': 'www.odoo.com',
+            'campaign_id': self.utm_campaign_3.id,
+            'medium_id': self.utm_medium.id,
+            'source_id': self.utm_source.id
+        })
+        self.link_tracker_click_1 = self.LinkTrackerClick.create({
+            'link_id': self.link_tracker_1.id
+        })
+        self.link_tracker_click_2 = self.LinkTrackerClick.create({
+            'link_id': self.link_tracker_2.id
+        })
+        self.link_tracker_click_3 = self.LinkTrackerClick.create({
+            'link_id': self.link_tracker_3.id
+        })
+        return result
+
+    def test_merge_unique(self):
```

Comment:
As method docstring, we sometimes put a small sentence explaining what the test does in summary:

> """ Merge 2 campaigns and check that everything is correctly cleaned up.
> Link trackers should be cleaned in this case since they're not unique anymore
> (see 'link_tracker.py'#_clean_duplicates) """

Try to make one for the other test too.